src/main/java/juja/microservices/teams/service/TeamService.java

```diff
@@ -1,7 +1,43 @@
 package juja.microservices.teams.service;
 
+import juja.microservices.teams.dao.TeamRepository;
+import juja.microservices.teams.entity.Team;
+import juja.microservices.teams.exceptions.TeamsException;
+import juja.microservices.teams.exceptions.UserInSeveralTeamsException;
+import juja.microservices.teams.exceptions.UserNotInTeamException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Service;
 
+import javax.inject.Inject;
+import java.time.LocalDateTime;
+import java.util.List;
+
+/**
+ * @author Ivan Shapovalov
+ */
 @Service
 public class TeamService {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Inject
+    private TeamRepository teamRepository;
+
+    public Team dismissTeam(String uuid) {
+        logger.debug("Send request to repository: get teams of user <{}> for current date", uuid);
+        List<Team> teams = teamRepository.getUserTeams(uuid);
+        logger.info("Received list of teams, size = {}", teams.size());
```

Comment:
If repository return teams as null then NullPointerException will occur in your log. Please be careful to invoke methods from unchecked results.